### ISSUE TITLE
Impossible to start preview of docusaurus website

### ISSUE THREAD

**Issue by FabienSe** [quarto-dev/quarto-cli#4312]
### Bug description

I am trying to preview a simple docusaurus website on windows 10 thought VSCode but I got the following error:

``` console
$quarto preview
Preparing to preview

Watching files for changes
ERROR: NotFound: program not found
```
I have to say I have no clue which program is not found.
I tried the following:
- Install chromium with `quarto tools install chromium`
- If I preview directly the website with `npm start`, it is working. So docusaurus installation seems fine.
- If I am tring to render/preview the files directly with the integrated button to VS code.
I got the same error:
```console
Preview service running (3985)
ERROR: NotFound: program not found
```
So the error seems linked to quarto.
- I tried to open the exact same setup in WSL and it is working. It is missing something on windows configuration then. But I cannot find what.
- If I am tring to remove `type: docusaurus` in _quarto.yml. It is working. So seems to be linked to quarto and docusaurus integration.
```yml
project:
  type: docusaurus
```

Can you help me with this error ? Thanks a lot.

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by cderv** [comment]
I'll try to reproduce. 

issue seems to happen in VScode right ? What is the version of your VS code extension ? 

Thanks

**Comment by cderv** [comment]
> Yes the issue is in VScode with the integrated terminal.

Just to clarify: Does it means it works outside of VSCODE ? Did you try it ? This would help focus on where the issue could be `quarto-cli` or its vscode extension

**Comment by cderv** [comment]
Thanks for all that information. I'll look into it.

**Comment by cderv** [comment]
So I can reproduce - here is the traceback 
````
> quarto preview
Preparing to preview

Watching files for changes
ERROR: NotFound: program not found

NotFound: program not found
    at opRun (deno:runtime/js/40_process.js:33:16)
    at Object.run (deno:runtime/js/40_process.js:116:17)
    at externalPreviewServer (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/project/serve/serve.ts:396:24)
    at serveProject (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/project/serve/serve.ts:269:13)
    at async Command.fn (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/command/preview/cmd.ts:309:9)
    at async Command.execute (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/vendor/deno.land/x/cliffy@v0.25.4/command/command.ts:1790:7)
    at async quarto (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/quarto.ts:111:3)
    at async file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/quarto.ts:138:5
````

We probably need to adjust another time our command to run the process required to launch docusaurus website on Windows. `Deno.run()` only is not working.

**Comment by cderv** [comment]
@FabienSe this should be fixed in next build of pre-release that should happen later in the day probably. 

Thank you for the report

**Comment by cderv** [comment]
This fix is not yet available in current latest 1.3.x but will be in next. and then you will need to install the pre-release to have the fix. 

> Is this version 1.3.X considered stable too ?

This is not stable as 1.2 is. This is a pre-release version- we build several time per day a version of Quarto based on github repo main state. If you start using 1.3 it is advise to update regularly. 

If you prefer to wait for next release, this should happen probably in March, so not too far away. There will be first a release canditate, and then new release.

**Comment by cderv** [comment]
FWIW `v1.3.197` is the first version with the fix
